packages/storybook/src/css/StatusBadge.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { Meta, StoryObj } from '@storybook/react';
+import readme from '../../../../components/StatusBadge/README.md';
+import { StatusBadge } from '@utrecht/component-library-react';
```

Comment:
Is het handig om deze dan die van Den Haag te maken omdat die wordt geexporteerd?